azure/durable_functions/models/TaskOrchestrationExecutor.py

```diff
@@ -276,6 +276,16 @@ def get_orchestrator_state_str(self) -> str:
             message contains in it the string representation of the orchestration's
             state
         """
+        if(self.output):
```

Comment:
nit: `self.output` could be set to `False`, and that would skip this if-statement despite needing to be serialized. So please check specifically for `self.output` being `None`.
```suggestion
        if(self.output is not None):
```

Reply:
and please E2E test this again after you incorporate it. Thanks!